code/iaas/model/src/main/java/io/cattle/platform/core/dao/impl/NetworkDaoImpl.java

```diff
@@ -72,4 +85,37 @@ public Nic getPrimaryNic(long instanceId) {
                         .and(NETWORK.REMOVED.isNull()))
                 .fetchInto(NetworkRecord.class);
     }
+
+    @Override
+    public Network getNetworkForObject(Object object) {
```

Comment:
After this is merged, I will use this method for native docker. I may need to add support for getting an unmanaged network (as opposed to assuming `KIND_HOSTONLY`).

Just bringing this up mainly for informational purposes.